### PR TITLE
Romanian i18n - refs #1521

### PR DESCRIPTION
**Pull request by alecghica**
refs #1521

### CODE REVIEW

File: locales/ro/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Add"
+msgstr "Adăugare"
+
+#: components/manage/Controlpanels/AddonsControlpanel
+msgid "Add Addons"
+msgstr "Adăugare add-on-uri"
+
+#: components/manage/Toolbar/Types
+msgid "Add Content"
+msgstr "Adăugați conținut"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:54]: Consistency. Either use `Adăugare` or `Adăugați`

File: locales/ro/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Back"
+msgstr "Înapoi"
+
+#: components/manage/Diff/Diff
+msgid "Base"
+msgstr "Baza"
+
+#: components/manage/Sidebar/Sidebar
+msgid "Block"
+msgstr "Blocare"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:169]: It's the object `Bloc` not the action :wink:

File: locales/ro/LC_MESSAGES/volto.po [hunk 3]
```diff
@@ -0,0 +1,1939 @@
...
+
+#: components/manage/Blocks/Listing/ListingSidebar
+msgid "Block style"
+msgstr "Blocare stil"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:173]: Noun `Block`

File: locales/ro/LC_MESSAGES/volto.po [hunk 4]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Both email address and password are case sensitive, check that caps lock is not enabled."
+msgstr "Atât adresa de e-mail, cât și parola sunt case-sensitive, verificați dacă tasta "Caps Lock" nu este activată."
+
+#: components/theme/Breadcrumbs/Breadcrumbs
+msgid "Breadcrumbs"
+msgstr "Breadcrumbs"
+
+#: components/manage/Contents/ContentsUploadModal
+msgid "Browse"
+msgstr ""
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:185]: Maybe `Răsfoiește`

File: locales/ro/LC_MESSAGES/volto.po [hunk 5]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Center"
+msgstr "Centrează"
+
+#: components/manage/History/History
+msgid "Change Note"
+msgstr "Schimbare notă"
+
+#: components/manage/Preferences/ChangePassword
+msgid "Change Password"
+msgstr "Schimbă parola"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:225]: Consistency for `Change`

File: locales/ro/LC_MESSAGES/volto.po [hunk 6]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Group"
+msgstr "Grup"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Group created"
+msgstr "Grup creat"
+
+#: components/manage/Controlpanels/UsersControlpanel
+msgid "Groupname"
+msgstr "Nume de grup"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:691]: `Numele grupului` or `Nume grup` would fit better.

File: locales/ro/LC_MESSAGES/volto.po [hunk 7]
```diff
@@ -0,0 +1,1939 @@
...
+msgstr "Imaginea de start"
+
+#: components/manage/Blocks/Maps/Edit
+#: helpers/AlignBlock/AlignBlock
+msgid "Left"
+msgstr "Stânga"
+
+#: components/manage/Toolbar/Toolbar
+msgid "Link"
+msgstr "Link"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:839]: `Legătură`

File: locales/ro/LC_MESSAGES/volto.po [hunk 8]
```diff
@@ -0,0 +1,1939 @@
...
+
+#: components/manage/Blocks/Listing/ListingSidebar
+msgid "Link more"
+msgstr ""
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:843]: I think it's about a link for more items

File: locales/ro/LC_MESSAGES/volto.po [hunk 9]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Link title"
+msgstr "Link Titlu"
+
+#: components/manage/Blocks/Image/ImageSidebar
+#: components/manage/Blocks/LeadImage/LeadImageSidebar
+#: components/manage/Blocks/Listing/ListingMore
+#: components/manage/Blocks/Maps/MapsSidebar
+#: components/manage/Blocks/Video/VideoSidebar
+msgid "Link to"
+msgstr ""
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:855]: ...

File: locales/ro/LC_MESSAGES/volto.po [hunk 10]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "Make the table compact"
+msgstr "Compactati tabelul"
+
+#: components/manage/Blocks/Maps/MapsSidebar
+msgid "Map"
+msgstr "Hartă"
+
+#: components/manage/Blocks/Maps/MapsSidebar
+msgid "Maps URL"
+msgstr ""
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:896]: `URL Hartă`

File: locales/ro/LC_MESSAGES/volto.po [hunk 11]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "html"
+msgstr "html"
+
+#: config/Blocks
+msgid "image"
+msgstr "imagine"
+
+#: components/theme/PasswordReset/RequestPasswordReset
+msgid "label_my_email_address_is"
+msgstr "eticheta_adresa_mea_de_email_este"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:1843]: Not sure about this one

File: locales/ro/LC_MESSAGES/volto.po [hunk 12]
```diff
@@ -0,0 +1,1939 @@
...
+msgid "event_when"
+msgstr ""
+
+#: components/theme/View/EventView
+msgid "event_where"
+msgstr ""
+
+#: components/theme/PasswordReset/RequestPasswordReset
+msgid "heading_sent_password"
+msgstr "titlu_parola_trimisa"
```
Comment (avoinea) [inline locales/ro/LC_MESSAGES/volto.po:1827]: Not sure about this one

File: locales/ro/LC_MESSAGES/volto.po (hunk 10 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 11 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po:1843]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 9 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 8 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 5 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 7 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po:839]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 6 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po:691]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 4 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 3 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 2 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

File: locales/ro/LC_MESSAGES/volto.po (hunk 1 above)
Comment (alecghica) [reply locales/ro/LC_MESSAGES/volto.po]: Fixed.

**Pull request merged**